ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -519,19 +542,27 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
 
         // "Real" as opposed to "Temp". Important we return the saved address as properties are added on server
         const saveRealAddressAction$ = this.saveTrigger$.pipe(
-            withLatestFrom(this.formErrorMessages$, currentAddress$),
-            filter(([_, formErrors, addressToSave]) => {
-                return !formErrors?.length && canSaveRealAddress(addressToSave);
+            withLatestFrom(currentAddress$, this.formErrorMessages$, this.verifyFieldErrors$, ignoreEasyPostErrorsObservable),
+            filter(([_, addressToSave, formErrors, fieldErrors, ignoreEasyPost]) => {
+                return !formErrors?.filter(error => !(ignoreEasyPost && error.isEasyPostError)).length
```

Comment:
allow to save address if ignoreEasyPost is checked and all errors are easy post errors